modules/adprimeBidAdapter.js

```diff
@@ -0,0 +1,172 @@
+
+  isBidRequestValid: (bid) => {
+    return Boolean(bid.bidId && bid.params && bid.params.placementId);
+  },
+
+  buildRequests: (validBidRequests = [], bidderRequest) => {
+    let winTop = window;
+    let location;
+    try {
+      location = new URL(bidderRequest.refererInfo.referer)
+      winTop = window.top;
+    } catch (e) {
+      location = winTop.location;
+      utils.logMessage(e);
+    };
+    let placements = [];
+    let request = {
+      deviceWidth: winTop.screen.width,
+      deviceHeight: winTop.screen.height,
+      language: (navigator && navigator.language) ? navigator.language.split('-')[0] : '',
+      secure: 1,
+      host: location.host,
+      page: location.pathname,
+      placements: placements
+    };
+    if (bidderRequest) {
+      if (bidderRequest.uspConsent) {
+        request.ccpa = bidderRequest.uspConsent;
+      }
+      if (bidderRequest.gdprConsent) {
+        request.gdpr = bidderRequest.gdprConsent
+      }
+    }
+    const len = validBidRequests.length;
+
+    for (let i = 0; i < len; i++) {
+      let bid = validBidRequests[i];
+      const { mediaTypes } = bid;
+      const placement = {};
+      let sizes
+      let identeties = {}
+      if (mediaTypes) {
+        if (mediaTypes[BANNER] && mediaTypes[BANNER].sizes) {
+          placement.adFormat = BANNER;
+          sizes = mediaTypes[BANNER].sizes
+        } else if (mediaTypes[VIDEO] && mediaTypes[VIDEO].playerSize) {
+          placement.adFormat = VIDEO;
+          sizes = mediaTypes[VIDEO].playerSize
+          placement.minduration = mediaTypes[VIDEO].minduration;
+          placement.maxduration = mediaTypes[VIDEO].maxduration;
+          placement.mimes = mediaTypes[VIDEO].mimes;
+          placement.protocols = mediaTypes[VIDEO].protocols;
+          placement.startdelay = mediaTypes[VIDEO].startdelay;
+          placement.placement = mediaTypes[VIDEO].placement;
+          placement.skip = mediaTypes[VIDEO].skip;
+          placement.skipafter = mediaTypes[VIDEO].skipafter;
+          placement.minbitrate = mediaTypes[VIDEO].minbitrate;
+          placement.maxbitrate = mediaTypes[VIDEO].maxbitrate;
+          placement.delivery = mediaTypes[VIDEO].delivery;
+          placement.playbackmethod = mediaTypes[VIDEO].playbackmethod;
+          placement.api = mediaTypes[VIDEO].api;
+          placement.linearity = mediaTypes[VIDEO].linearity;
+        } else {
+          placement.adFormat = NATIVE;
+          placement.native = mediaTypes[NATIVE];
+        }
+      }
+      if (bid.userId && bid.userId.idl_env) {
+        identeties.identityLink = bid.userId.idl_env
+      }
+
+      placements.push({
+        ...placement,
+        placementId: bid.params.placementId,
+        bidId: bid.bidId,
+        sizes: sizes || [],
+        wPlayer: sizes ? sizes[0] : 0,
+        hPlayer: sizes ? sizes[1] : 0,
+        schain: bid.schain || {},
+        keywords: bid.params.keywords || [],
+        audiences: bid.params.audiences || [],
```

Comment:
Your docs doesn't have this parameter.  
Please open PR on docs repo to include `audiences` parameter.

Reply:
doc - https://github.com/prebid/prebid.github.io/pull/3176